libfmt 12.2.0 breakage

After updating libfmt in pkgsrc to 12.2.0, volk 3.3.0 stopped building with:
```
pbulk/work/math/volk/work/volk-3.3.0/lib/qa_utils.cc: In function 'void print_error_table(const std::vector<unsigned int>&, const std::vector<void*>&, const std::vector<volk_type_t>&, void*, void*, const volk_type_t&, float, int)':
/pbulk/work/math/volk/work/volk-3.3.0/lib/qa_utils.cc:626:38: error: 'format' is not a member of 'fmt'
  626 |         fmt::print(" | {:>10}", fmt::format("in{}", k));
      |                                      ^~~~~~
/pbulk/work/math/volk/work/volk-3.3.0/lib/qa_utils.cc: In lambda function:
/pbulk/work/math/volk/work/volk-3.3.0/lib/qa_utils.cc:1454:21: error: 'format' is not a member of 'fmt'
 1454 |         return fmt::format("{:.2f} ms", ms);
      |                     ^~~~~~
/pbulk/work/math/volk/work/volk-3.3.0/lib/qa_utils.cc: In lambda function:
/pbulk/work/math/volk/work/volk-3.3.0/lib/qa_utils.cc:1458:21: error: 'format' is not a member of 'fmt'
 1458 |         return fmt::format("{:.1f} MB/s", mbps);
      |                     ^~~~~~
... (many more similar lines)
```
This patch fixes the problem:
```
--- lib/qa_utils.cc.orig        2026-06-25 07:29:56.366917249 +0000
+++ lib/qa_utils.cc
@@ -28,7 +28,7 @@
 #include <random>
 #include <vector> // for vector, _Bit_refe...

-#include <fmt/core.h>
+#include <fmt/format.h>

 // Warmup time for CPU frequency scaling (ms)
 static double g_warmup_ms = 2000.0;
```

--- a/lib/qa_utils.cc
+++ b/lib/qa_utils.cc
@@ -28,7 +28,7 @@
 #include <random>
 #include <vector> // for vector, _Bit_refe...
 
-#include <fmt/core.h>
+#include <fmt/format.h>
 
 // Warmup time for CPU frequency scaling (ms)
 static double g_warmup_ms = 2000.0;

